README.md

```diff
@@ -15,11 +15,14 @@ Frequently asked questions are available on the [wiki](https://github.com/NVIDIA
 
 ## Getting Started
 
-**Make sure you have installed the [NVIDIA driver](https://docs.nvidia.com/datacenter/tesla/tesla-installation-notes/index.html) and Docker engine for your Linux distribution**.
+On the host system:
+1. Install the [NVIDIA driver](https://docs.nvidia.com/datacenter/tesla/tesla-installation-notes/index.html) and Docker engine for your Linux distribution.
+2. Install the NVIDIA Container Toolkit, following the [installation guide](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html).
```

Comment:
This point is incorrect. The `nvidia-docker2` package is provided as a simple wrapper for docker users and is not the package for the NVIDIA Container Toolkit.

Apart from the `nvidia-docker` wrapper that is installed the `nvidia-docker2` package overwrites the Docker `/etc/docker/daemon.json` to include the `nvidia` runtime. This is being deprecated in favour of using a CLI tool to configure this.

Running:
```
sudo nvidia-ctk runtime configure --runtime=docker
```
(assuming the `nvidia-container-toolkit-base` package has been installed)

will update the docker config without simply overwriting it. The docker daemon still needs to be restarted.